src/reconcile/rec_write.c

```diff
@@ -3380,6 +3380,9 @@ __rec_write_wrapup(WT_SESSION_IMPL *session, WTI_RECONCILE *r)
         mod->mod_multi_entries = r->multi_next;
         mod->rec_result = WT_PM_REC_MULTIBLOCK;
 
+        if (r->multi_next > 1)
+            F_SET(ref, WT_REF_FLAG_REC_SPLIT);
```

Comment:
`WT_REF_FLAG_REC_SPLIT ` -> `WT_REF_REC_SPLIT `

Reply:
Do we really need a ref flag for this? Is there any other location we can save this information? The split information is available in the `page->modify`, why can't we save it there itself with another variable?

Reply:
That information is lost after a new reconciliation or when the child is evicted.

Reply:
we need the information of what the last internal page reconciliation has written for this ref.